articles/azure-monitor/insights/container-insights-enable-new-cluster.md

```diff
@@ -29,8 +29,8 @@ You can enable monitoring of an AKS cluster using one of the supported methods:
 To enable monitoring of a new AKS cluster created with Azure CLI, follow the step in the quickstart article under the section [Create AKS cluster](../../aks/kubernetes-walkthrough.md#create-aks-cluster).  
 
 >[!NOTE]
->If you choose to use the Azure CLI, you first need to install and use the CLI locally. You must be running the Azure CLI version 2.0.59 or later. To identify your version, run `az --version`. If you need to install or upgrade the Azure CLI, see [Install the Azure CLI](https://docs.microsoft.com/cli/azure/install-azure-cli). 
->
+>If you choose to use the Azure CLI, you first need to install and use the CLI locally. You must be running the Azure CLI version 2.0.74 or later. To identify your version, run `az --version`. If you need to install or upgrade the Azure CLI, see [Install the Azure CLI](https://docs.microsoft.com/cli/azure/install-azure-cli). 
+>If you have installed the aks-preview extension, please make sure to remove any changes (if any) that you've made in it, since it can override the default Azure CLI behavior.
```

Comment:
I had to look up what the AKS preview extension was, and this sentence needs to be reworded for consistency with AKS docs regarding that extension. Suggest rewording to - "If you have installed the aks-preview *CLI extension* version 0.4.12 or higher, remove any changes you have made using it as it can override the default Azure CLI behavior."  Can we be clearer though about "changes" (provide example), and when we  say "override  the default Azure CLI behavior, what does that mean exactly")?

Reply:
AKS Public preview features are not supported in Fairfax. For e.g. if you've installed the aks-preivew extension and are using a public preview feature like mentioned here -> https://github.com/Azure/AKS/blob/master/previews.md#preview-features then the cluster creation or monitoring addon enablement will fail because the preview features aren't supported in Fairfax.

I can get in a call and explain this over Teams if I'm not explaining it properly. 

Currently updating it to the statement you posted in the comment with a little edit:

If you have installed the aks-preview CLI extension version 0.4.12 or higher, remove any changes you have made to enable a preview extension as it can override the default Azure CLI behavior since AKS Preview features aren't available in Az US Governmnet cloud.

I can update it to what you think is more clear.